docs/learn/learn-nft-pallets.md

```diff
@@ -150,9 +189,18 @@ Creating an item usually involves setting some attributes specific to that item.
 - Take ownership of an NFT from the [pallet-nfts](#nfts-pallet)
 - Create a new asset in [pallet-assets](https://polkadot.js.org/docs/substrate/extrinsics#assets)
 - Mint the input amount to the previous owner of the NFT as the beneficiary
+- Mass minting: Minting multiple items in one single transaction. Ths will require the user to
+  provide a .csv file with two columns: NFT ID and CID if metadata.
 
 ## Uniques Pallet
 
+:::info
+
+The Uniques Pallet is being decommissioned. Everything related to NFTs will be covered by the the
```

Comment:
deprecated
decommissioned is a strong word I think in this case, we are not removing it (at least not yet)